pkg/sfu/buffer/buffer.go

```diff
@@ -31,19 +31,19 @@ type pendingPackets struct {
 }
 
 type ExtPacket struct {
-	Head     bool
-	Cycle    uint32
-	Arrival  int64
-	Packet   rtp.Packet
-	Payload  interface{}
-	KeyFrame bool
+	Head       bool
+	Arrival    int64
+	Packet     rtp.Packet
+	Payload    interface{}
+	KeyFrame   bool
+	RawPayload []byte
```

Comment:
Relaydown will send RawPayload directly

Reply:
Nice!!!

Maybe call it `RawPacket` as there is `Payload` inside `rtp.Packet` which does not include the RTP header bytes. Just to make it clear, suggesting this change.

Reply:
sounds good. renamed